webpack.config.js

```diff
@@ -21,4 +21,10 @@ module.exports = {
   resolve: {
     extensions: ['.js'],
   },
+  externals: {
```

Comment:
This externals are needed to resolve the issue of having one version of react across the main app and this component package.